### DISCUSSION TITLE
I havent recieven educational benefits

### DISCUSSION THREAD

**Discussion by Jesse-Voo** [community/community#129841] [GitHub Education]
### Hi everyone,

I would like to recieve these, because I need them for a project!

**Comment by davevad93** [comment]
Hi @Jesse-Voo ,

It may take a while, from a few days to a few weeks, depending on different factors such as the volume of applications. When the benefits become available, the status of the application will change to:

`Congrats! Your request for @Jesse-Voo  was approved and your benefits are now available`
